src/server/utils/auth.js

```diff
@@ -1,226 +0,0 @@
-const _ = require('lodash');
```

Comment:
So, does Passport handle the FHIR server end of the transaction, the OAuth server end, or both?

Reply:
both